tools/generator/templates/includes/Makefile.language.jinja2

```diff
@@ -0,0 +1,48 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (c) 2020 Association Prologin <association@prologin.org>
+
+# Add all your champion files in here, separated by spaces
+# Include all the sources, headers and README files.
```

Comment:
I’m affraid if we don’t specify the extention here, the candidate may use extensions like .cpp because they prefere it, and the makefile wo’nt handle those.